### DISCUSSION TITLE
Make own footsteps quietter

### DISCUSSION THREAD

**Discussion by AdamTadeusz** [NeotokyoRebuild/neo#663] [Ideas]
Do you guys think it would be a good addition to the game to make a player's own footsteps quieter? From the limiting testing I did this doesn't seem to be the case in the original, but might be a great addition. There's various studies that suggest your brain filters out sounds you make yourself, in a video game however where you do not have a great feel of your own body (Your legs are invisible in first person and you can't feel when they will make a step, not that footsteps are strictly tied to when the model animation makes a step anyway) its far more difficult to make that connection.

Helping players filter out their own footsteps by making them quieter would help combat camping as players could play more aggressively while still maintaining a higher level of situational awareness.

Here's a video of a player walking around with another bot to their side. The player's own footstep volume has been reduced ten-fold.

https://github.com/user-attachments/assets/d88bb6a7-f5cd-430b-8901-8aa431b1363c

**Comment by xedmain** [comment]
havent watched the video yet, but what about your own footsteps having a different pitch?

them being quieter may make yourself think that theres someone either silently walking towards you or someone walking far away if youre A D A D A D spamming or the like (need to test ig?)

**Comment by kassibuss** [comment]
Wouldn't mind, but would need testing

**Comment by Rainyan** [comment]
Not necessarily against, but this could make it unintuitive for new players to accurately estimate how much noise they're making. And when you need to go silent, missing out on hearing a stray footstep you accidentally emitted could lead to making the wrong play, incorrectly assuming you were undetected. This could maybe also come into play when trying to keep track of whether your bhop made a noise?

As a cvar slider, I could see it being uncontroversial.
